content/english/python-basics/basics/variables.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: "Variables"
-date: 2020-02-10T13:24:17-07:00
+date: 2022-09-19T14:45:38-07:00
 draft: false
 weight: 4
 --- 
 
-Variables are simply names that we can give to values such as strings, numbers, and booleans. Here’s how to make a variable named s. We say s is a string that has the value `"Hello, World!"`. Can you describe the following variables?
+In a previous exercise, we learned to print different statements with `print("")`. While it is great to print out a number or a sentence, we haven't given them a meaning. Variables are simply names that we can give to values such as strings, numbers, and booleans. Here’s how to make a variable named s in python. We say s is a string that has the value `"Hello, World!"`. 
```

Comment:
Consistently use Python instead of python

```suggestion
In a previous exercise, we learned to print different statements with `print("")`. While it is great to print out a number or a sentence, we haven't given them a meaning. Variables are simply names that we can give to values such as strings, numbers, and booleans. Here’s how to make a variable named s in Python. We say s is a string that has the value `"Hello, World!"`. 
```